cmd/fcaddr/main.go

```diff
@@ -0,0 +1,92 @@
+package main
+
+import (
+	"encoding/hex"
+	"flag"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/filecoin-project/go-address"
+
+	"golang.org/x/xerrors"
+)
+
+const (
```

Comment:
Not: consider defining theses in constant.go

Reply:
I can put them in a constant.go file in this folder, but I wouldn't want to put them in the top level as they don't apply to the library itself but this small little application. They are strings here so they can be accepted as input on the CLI.

Reply:
Sorry that was supposed to be a "Nit", I think what you've done here is fine, I was just being picky. :shipit: